app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,16 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+.navbar-default {
+    background-color: #8bc9ee;
```

Comment:
Color literals like `#8bc9ee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces